app/src/main/java/com/example/androidhomework/ComposeView.kt

```diff
@@ -0,0 +1,261 @@
+package com.example.androidhomework
+
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.*
+import com.example.androidhomework.R.drawable.*
+import java.time.format.TextStyle
+
+
+@OptIn(ExperimentalUnitApi::class)
+@Composable
+
+internal fun CorpseBrideScreen() {
+
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState())
+            .background(Color.White)
+    ) {
+
+      Box(
+          modifier = Modifier.fillMaxWidth(),
+          contentAlignment = Alignment.BottomStart
+
+      ) {
+          Image(
+              painter = painterResource(id = ic_header_image),
+              contentDescription = ""
+          )
+
+          Text(
+              text = stringResource(id = R.string.movie_title),
+              color = Color.White,
+              fontSize = TextUnit(24f, TextUnitType.Sp),
+              fontWeight = FontWeight.Bold,
+              modifier = Modifier
+                  .fillMaxWidth()
+                  .padding(
+                      start = dimensionResource(id = R.dimen.default_margin),
+                      end = dimensionResource(id = R.dimen.default_margin),
+                      bottom = dimensionResource(id = R.dimen.default_margin)
+                  )
+
+          )
+      }
+        Row(
+            horizontalArrangement =  Arrangement.SpaceBetween
+        ) {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .wrapContentHeight()
+                    .padding(12.dp),
```

Comment:
I think this should be `.padding(top = 12.dp)` 
If it's just 12.dp then that padding will be applied to all sides + you have 16.dp on start in next LabelView 
That means that LabelView would be moved from parent on start for 28.dp